compiler-rt/lib/asan/asan_flags.cpp

```diff
@@ -144,7 +144,8 @@ static void InitializeDefaultFlags() {
   DisplayHelpMessages(&asan_parser);
 }
 
```

Comment:
please extract this rename into a separate PR

Reply:
Removed: https://github.com/llvm/llvm-project/pull/122990/commits/d45c846c07946671a5b865bd0bbf54cd29707e95